iina/SleepPreventer.swift

```diff
@@ -29,7 +37,10 @@ class SleepPreventer: NSObject {
     if success == kIOReturnSuccess {
       preventedSleep = true
     } else {
-      Utility.showAlert("sleep")
+      Logger.log("Cannot prevent display sleep: \(errorToString(success)) (\(success))", level: .error)
```

Comment:
Honestly we should just force unwrap this. If it fails, then something has gone really wrong and we should know about it.

```suggestion
      Logger.log("Cannot prevent display sleep: \(String(cString: mach_error_string(success))) (\(success))", level: .error)
```

Reply:
Fixed.